bindings/python/tests/test_triplestore.py

```diff
@@ -0,0 +1,98 @@
+from dlite.triplestore import en, Literal, Triplestore, RDF, RDFS, XSD, OWL
+
+
+# Test namespaces
+assert str(RDF) == "http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+assert RDF.type == "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+
+# Test RDF literal
+l1 = Literal("Hello world!")
+assert l1 == "Hello world!"
+assert isinstance(l1, str)
+assert l1.lang is None
+assert l1.datatype is None
+assert l1.toPython() == "Hello world!"
+assert l1.value == "Hello world!"
+assert l1.n3() == '"Hello world!"'
+
+l2 = Literal("Hello world!", lang="en")
+assert l2.lang == "en"
+assert l2.datatype == None
+assert l2.value == "Hello world!"
+assert l2.n3() == '"Hello world!"@en'
+
+l3 = en("Hello world!")
+assert l3.n3() == '"Hello world!"@en'
+
+l4 = Literal(42)
+assert l4.lang == None
+assert l4.datatype == XSD.integer
+assert l4.value == 42
+assert l4.n3() == f'"42"^^{XSD.integer}'
+
+l5 = Literal(42, datatype=float)
+assert l5.lang == None
+assert l5.datatype == XSD.double
+assert l5.value == 42.0
+assert l5.n3() == f'"42"^^{XSD.double}'
+
+# Test rdflib triplestore backend
```

Comment:
Again, this comment does not give us much information. What are we testing in the backend, how are we testing it, why are we testing in the way that we are?